app.js

```diff
@@ -92,7 +92,9 @@ const csp = {
       'api.geocode.earth',
       'downloads.mailchimp.com.s3.amazonaws.com',
       'checkout.stripe.com',
-      'plausible.io'
+      'plausible.io',
+      'cdn.coil.com',
+      "'unsafe-inline'"
```

Comment:
Since we're rolling back the script additions in the template, we should be able to remove the `unsafe-inline` directive, as well.